airmozilla/manage/templates/manage/regions.html

```diff
@@ -0,0 +1,51 @@
+{% extends 'manage/manage_base.html' %}
```

Comment:
I made a small PR towards your PR on this https://github.com/bugzPDX/airmozilla/pull/1